source/FreeplayState.hx

```diff
@@ -44,7 +44,8 @@ class FreeplayState extends MusicBeatState
 		for (i in 0...initSonglist.length)
 		{
 			var data:Array<String> = initSonglist[i].split(':');
-			songs.push(new SongMetadata(data[0], Std.parseInt(data[2]), data[1]));
+			if(Std.parseInt(data[2]) <= FlxG.save.data.weekUnlocked - 1)
+				songs.push(new SongMetadata(data[0], Std.parseInt(data[2]), data[1]));
```

Comment:
maybe make it show a lock icon on the song and not let u enter it instead of just straight up not showing it but if ur lazy and dont care then this is fine

Reply:
maybe I'll convert this to a Enhancement request when this gets merged